conda_concourse_ci/bootstrap/config/config.yml

```diff
@@ -10,6 +10,7 @@ intermediate-server: your-intermediate-server
 intermediate-config-folder: your-intermediate-base-folder/base-name/config
 intermediate-recipe-folder: your-intermediate-base-folder/base-name/plan_and_recipes
 intermediate-artifacts-folder: your-intermediate-base-folder/base-name/artifacts
+intermedia-source-folder: your-intermediate-base-folder/base-name/source
```

Comment:
base-name is the repo.  I don't think we want per-repo storage of source.  I think you should just get rid of the middle base-name here.